suzieq/poller/services/interfaces.py

```diff
@@ -217,6 +217,8 @@ def _clean_cumulus_data(self, processed_data, raw_data):
 
             if entry['type'] == 'ether':
                 entry['type'] = 'ethernet'
+            if entry.get('type', '') == 'vxlan':
+                entry['speed'] = 0
```

Comment:
I think it's better to use `NO_SPEED` or the function `_get_missing_speed_value()`
```suggestion
                entry['speed'] = NO_SPEED
```